tests/rados/test_scrub_chunk_max.py

```diff
@@ -0,0 +1,173 @@
+"""
+Module to verify funationality and effect of OSD scrub_chunk_max parameter.
+BZ #1382226 - PG scrub bypasses 'osd_scrub_chunk_max' limit to find hash boundary
+"""
+import datetime
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    # CEPH-10792
+    Test to verify PG scrub is within the 'osd_scrub_chunk_max' limit
+    1. Create a replicated pool with single pg
+    2. Grab the default value of 'osd_scrub_chunk_max'
+    3. Set 'osd_scrub_chunk_max' to a low value, preferably lower than
+    the default. e.g. 5
+    4. Write few objects(>osd_scrub_chunk_max) to the created pool
+    5. Start deep-scrub of the pool with 1 pg having few objects
+    6. Wait for deep-scrub to complete and verify primary OSD logs
+        expected to find logs having entries like below:
+            be_scan_list (0/5 5:18dee445:::obj-12:head deep)
+            be_scan_list (0/5 5:18dee445:::obj-12:head deep)
+    7. Delete the created pools
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    rhbuild = config["rhbuild"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_obj)
+    pool_obj = PoolFunctions(node=cephadm)
+    client_node = ceph_cluster.get_nodes(role="client")[0]
+    installer_node = ceph_cluster.get_nodes(role="installer")[0]
+    pg_scrub_log_list = []
+    pool_name = "test_scrub_max"
+    log.info(
+        "Running test case to verify PG scrub is within the 'osd_scrub_chunk_max' limit"
+    )
+
+    try:
+        # create a replicated pool with single pg
+        rados_obj.create_pool(pool_name=pool_name, **{"pg_num_max": 1})
+
+        # retrieve default value of osd_scrub_chunk_max
+        org_scrub_chunk_max = mon_obj.get_config(
+            section="osd", param="osd_scrub_chunk_max"
+        )
+        log.info(f"Original value of osd_scrub_chunk_max is {org_scrub_chunk_max}")
+        assert int(org_scrub_chunk_max) > 5
```

Comment:
Please remove the check. It's not a test failure if the value of 'org_scrub_chunk_max' is not greater than 5.

Reply:
done